exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,21 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  return num < 0;
+}
 
+function isBetween5and10(num) {
+  return num > 5 && num < 10;
+}
+
+function isShortName(str) {
+  return str.split(" ").length === 1;
+}
+
+function startsWithD(str) {
+  return str[0] === "D";
```

Comment:
Would do you think would happen here if an empty string was passed to this function (i.e. `startsWithD("")`)? How could you account for this?

Reply:
Thank you, done.